API/projectClass/customFileClass.m

```diff
@@ -318,9 +334,17 @@ function displayTable(obj)
         end
 
         function obj = setFunctionName(obj,whichCustom,name)
-            % Set a new filename
+            % Set a new function name
             obj.varTable{whichCustom,3} = {name};  
         end
```

Comment:
I think these one-liner functions that are called once should be removed with the line included in setCustomFile instead.